packages/trees/frontend/sirius-components-trees/src/views/TreeView.types.ts

```diff
@@ -19,6 +19,7 @@ export interface TreeViewComponentProps extends WorkbenchViewComponentProps {
   showToolBar: boolean;
   enableMultiSelection: boolean;
   treeOptions: TreeOptions | undefined;
+  markedItemId?: string[] | undefined;
```

Comment:
`markedItemIds` instead of `markedItemId`

Reply:
Done